package.json

```diff
@@ -110,6 +110,7 @@
     "prettier": "prettier \"**/*.{css,scss,json,md,html,yml}\" --ignore-path=.eslintignore",
     "start": "react-scripts start",
     "test": "npm run test:app",
+    "test:all": "npm run test:typecheck && npm run test:code && npm run test:other && npm run test:app -- --watchAll=false",
```

Comment:
Just out of curiosity.. but isn't the `test` suppose to run all these tests? At the moment for `test` we can just run `test:app`

Reply:
I think that we should retain `npm test` to start test runner (or something) because that's what everyone can use without needing to read `package.json`.

Reply:
I would assume `test` to run all the tests.. not just some random one.

Reply:
If someone want something special.. needs to read `package.json`

Reply:
`test` current starts test runner that allows you to run all tests or whatever you want - it doesn't run a random test.

So you'd make `npm test` run `test:all`?